CMakeLists.txt

```diff
@@ -1198,7 +1192,7 @@ add_library(mixxx-lib STATIC EXCLUDE_FROM_ALL
   src/widget/wwidgetgroup.cpp
   src/widget/wwidgetstack.cpp
 )
-target_precompile_headers(mixxx-lib PUBLIC
+set(MIXXX_PRECOMPILED_HEADER
```

Comment:
I thing we can introduce a different recompile header for different libraries.

Reply:
Yeah, go shout. It looks like the QML target only need `assert.h` anyway so I have introduced a `common` header list with just that one for now.

At first, I tried experiencing with a single PCH shared across the two targets but this didn't work.